apis/duck/cached_test.go

```diff
@@ -71,7 +71,7 @@ func TestSameGVR(t *testing.T) {
 	const iter = 10
 	for i := 0; i < iter; i++ {
 		errGrp.Go(func() error {
-			_, _, err := cif.Get(gvr)
+			_, _, err := cif.Get(context.Background(), gvr)
```

Comment:
I suppose this could use the context returned by `errGroup.WithContext` above to be a bit more "correct"?

Reply:
done in 77678bb83f220dd422bf886e7ce009f6f23d6616